src/views/InProgress.vue

```diff
@@ -522,7 +522,7 @@ export default defineComponent({
       const shipmentIdResps = await Promise.all(requestParams.map((params) => UtilService.findShipmentIdsForOrders(params.picklistBinIds, params.orderIds)))
       return Object.assign({}, ...shipmentIdResps)
     },
-    async reportIssue(order: any, itemsToReject: any) {
+    async reportIssue(issueOrder: any, itemsToReject: any) {
```

Comment:
Since, the function already suggest that we are reporting an issue, order parameter seems fine I think
```suggestion
    async reportIssue(order: any, itemsToReject: any) {
```